database/java/2017/5811.yaml

```diff
@@ -0,0 +1,46 @@
+---
+cve: CVE-2017-5811
+title: CVE in com.hpe.nv:hpe-nv-dtos
+description: >
+    A remote code execution vulnerability in HPE Network Automation version 9.1x, 9.2x, 10.0x, 10.1x and 10.2x were found.
+cvss_v2: 7.8
+references:
+    - http://www.securityfocus.com/bid/98331
+    - http://www.securitytracker.com/id/1038407
+    - https://support.hpe.com/hpsc/doc/public/display?docId=emr_na-hpesbgn03740en_us
+
+affected:
+    - groupId: com.hpe.nv
+      artifactId: hpe-nv-dtos
+      version:
+        - "1.0.0"
+      fixedin:
+        - "!FixMe"
```

Comment:
Where do we get this info? Do we leave it as it is if this vulnerability is not fixed yet?

Reply:
Exactly, if the vulnerability is not fixed yet, then we would remove the `fixedin` section completely. In this case, one of the [links in references](https://support.hpe.com/hpsc/doc/public/display?docId=emr_na-hpesbgn03740en_us) describes which versions have the vulnerabilities fixes ("RESOLUTION" section).